spec/forms/signups/create_form_spec.rb

```diff
@@ -84,24 +84,23 @@
           }
         end
         let(:enriched_address) do
-          Incognia::Address::Structured.new(
-            **address.merge(locale: described_class::EN_US_LOCALE)
-          )
+          address.merge(locale: described_class::EN_US_LOCALE)
         end
 
         it "also stores the address along with the signup" do
           created_signup = submit
 
           expect(
             created_signup.address.deep_symbolize_keys
-          ).to eq(structured_address: address)
+          ).to eq(structured_address: enriched_address)
         end
 
-        it 'requests Incognia w/ installation_id and address w/ default locale' do
-          allow(adapter).to receive(:register_signup) do |args|
-
+        it 'register signup w/ installation_id and address w/ default locale' do
+          allow(Signups::Register).to receive(:call) do |args|
```

Comment:
Shouldn't this allow be an expect?

Reply:
Good catch!

191c77a44e15d6926facc8cd01f1361facc9e3c1